### PR TITLE
Fall 17 Design

### PR DESCRIPTION
**Pull request by casey-chow**
Blocked by https://github.com/hackprinceton/styles/pull/1.

Please review ASAP for the following:

- Correct layout on any and every device you have
  - in **both** portrait and landscape orientations
  - if a computer, **all** browsers you have installed (specifically looking for IE/Edge, Opera, and Firefox)
- Code readability and maintainability
- Site readability

I will keep an ngrok to my computer while I am awake, ping me if it goes down: https://a320ad52.ngrok.io

### Devices/Browsers Tested

Please check off when you test.

**Desktop**
- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] IE/Edge

**Mobile**
- [x] Safari
- [x] Chrome Android
- [x] Firefox Android

### CODE REVIEW

**Comment by casey-chow** [comment]
Am currently blocked on the front page large layout, awaiting design to give me the correct assets.

**Comment by casey-chow** [comment]
Unblocked, and fixed the tree. PTAL

**Comment by rfblue2** [comment]
LG on Firefox

**Comment by casey-chow** [comment]
Found a visual bug on iPhone:

img_2200

**Comment by rfblue2** [comment]
Hmm, now when I shrink the window and view FAQs I see this:
image

**Comment by casey-chow** [comment]
@rfblue2 Fixed.

**Review by rfblue2 — APPROVED** [review]
I've tested with Firefox and Chrome Android.  LGTM

**Comment by casey-chow** [comment]
all good to go, pending content and director approval @dfan97

**Review by selina — APPROVED** [review]
IE/Edge looks good to me

**Comment by casey-chow** [comment]
@elizabethtian @dfan97 ready to merge, pending approval

File: index.html
Comment (zacharyliu) [inline index.html]: The trust badge isn't clickable since it's showing up under the `<section>`. Adding a z-index fixes it. Remember to update this to the 2018 badge/link.

File: _sass/index.scss
Comment (zacharyliu) [inline _sass/index.scss]: These dividing seem a little too aggressive, maybe try a lighter shade? There's no design in the spec for this section, but there's also no other place in the design where these dividers are shown.

File: _layouts/default.html
Comment (zacharyliu) [inline _layouts/default.html:79]: GitHub isn't letting me leave a line comment, but the Apply Now / Login links below probably also need the `large-horizontal` class

File: js/main.js
Comment (zacharyliu) [inline js/main.js:2]: This offset should be kept, since otherwise the menu links are slightly off

File: travel.html
Comment (zacharyliu) [inline travel.html:25]: Probably should delete this section

**Review by zacharyliu — COMMENTED** [review]
Should eventually update the repo link in the footer to point to the new static site repo, although the old link still redirects.

**Comment by zacharyliu** [comment]
Layout seems cramped on iPhone 5 sized screens (tested in Chrome emulator)

image image

File: js/main.js
Comment (casey-chow) [reply js/main.js:2]: Done, although `.top-bar` is variable height on mobile. Changed to `.title-bar`.

Comment (casey-chow) [reply js/main.js:2]: Done, although `.top-bar` is variable height on mobile. Changed to `.title-bar`.

File: _sass/index.scss
Comment (casey-chow) [reply _sass/index.scss]: I switched to orange. PTAL

File: index.html
Comment (casey-chow) [reply index.html]: I thought we're still in the 2017 season though?

File: _layouts/default.html
Comment (casey-chow) [reply _layouts/default.html:79]: 👍

File: index.html
Comment (zacharyliu) [reply index.html]: I think the seasons are based on the school year since the 2018 page has fall events in it

File: travel.html
Comment (casey-chow) [reply travel.html:25]: @dfan97 @elizabethtian

**Comment by dfan** [comment]
Reminder to add MLH Season '18's badge to the site.

File: travel.html
Comment (elizabethtian) [reply travel.html:25]: We can leave line 22, delete Routes and Schedules since those will change.

**Comment by casey-chow** [comment]
> Reminder to add MLH Season '18's badge to the site.

Done, commented for now.

**Pull request merged**
